src/invoices/Voucher/VoucherEditForm/VoucherEditForm.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'redux-form';

```

Comment:
@aliaksei-chumakou, should it be `react-final-form`?

Reply:
correct, we use react-final-form for new code